nimble/controller/src/ble_ll_adv.c

```diff
@@ -105,6 +105,7 @@ struct ble_ll_adv_sm
     uint8_t initiator_addr[BLE_DEV_ADDR_LEN];
     struct os_mbuf *adv_data;
     struct os_mbuf *scan_rsp_data;
+    struct os_mbuf *updated_data;
```

Comment:
not 100% sure, but this probably should be reset somewhere if adv is stopped before data were updated

Reply:
not really reset but updated. done